runtime/interface.go

```diff
@@ -82,6 +83,8 @@ type Interface interface {
 	AddAccountKey(address Address, publicKey *PublicKey, hashAlgo HashAlgorithm, weight int) (*AccountKey, error)
 	// GetAccountKey retrieves a key from an account by index.
 	GetAccountKey(address Address, index int) (*AccountKey, error)
+	AccountKeysCount(address Address) *big.Int
```

Comment:
Return `uint64` instead. It's pretty much impossible an account will have more keys than that